src/commands/cloud-run/flare.ts

```diff
@@ -0,0 +1,252 @@
+import IService = google.cloud.run.v2.IService
+import fs from 'fs'
+import path from 'path'
+import process from 'process'
+import util from 'util'
+
+import {ServicesClient} from '@google-cloud/run'
+import {google} from '@google-cloud/run/build/protos/protos'
+import chalk from 'chalk'
```

Comment:
Also, how does the first import work?
```suggestion
import IService = google.cloud.run.v2.IService
import fs from 'fs'
import path from 'path'
import process from 'process'
import util from 'util'
import chalk from 'chalk'

import {ServicesClient} from '@google-cloud/run'
import {google} from '@google-cloud/run/build/protos/protos'

```

Reply:
ESLint is telling me to put the `chalk` import under the `google` import